intelligent-response/nima-saei/colorpicker/functions/color.js

```diff
@@ -0,0 +1,1191 @@
+
```

Comment:
If you have used any opensource code here, it's better to add the reference here.

Reply:
It seems to be a obsolete code and better to be improved. And you could not bring all parts